src/components/views/Game.js

```diff
@@ -83,19 +89,49 @@ const Game = () => {
     }
   };
 
-  const createLink = () => {
-    //needs to be implemented
-    setCreated(true);
-    const link = "www.bg.com/game/" + gameName;
-    setGameLink(link);
+  const doLobby = async () => {
+    try {
+      const response = await createLobby(user.id);
+      setCreated(true);
+      const data = response.data;
+      setCreateCode(data.lobbyUuid);
+    } catch (error) {
+      alert(
+        `Something went wrong while creating a lobby: \n${handleError(error)}`
+      );
+    }
   };
 
   const copyLink = (text) => {
     navigator.clipboard.writeText(text);
     setCopied(true);
   };
 
-  const user = () => {
+  const enterLobby = async (lobbyId) => {
+    try {
+      await joinLobby(lobbyId, user.id, () => {
+        history.push("/board/" + lobbyId);
+      });
+    } catch (error) {
+      alert(
+        `Something went wrong while joining the lobby: \n${handleError(error)}`
+      );
+    }
+  };
+
+  const connectToWebsocket = async (lobbyId) => {
```

Comment:
First the connection to the lobby is made with a rest API call and then we connect to the websocket endpoint (which is not finished in the backend yet).